src/common/njord.ts

```diff
@@ -1071,3 +1075,7 @@ export const throwUserTransactionError = async ({
 export const coresToUsd = (cores: number): number => {
   return cores / 100; // 100 Cores = 1 USD
 };
+
+export const usdToCores = (usd: number): number => {
+  return usd * 100; // 1 USD = 100 Cores
```

Comment:
usd could be non integer so we need to make sure Cores are always integer

Reply:
Right. Yeah, let's do the removal of decimal places here.